Dockerfile

```diff
@@ -1,2 +1,30 @@
 FROM kylemanna/openvpn
 
+RUN wget https://storage.googleapis.com/kubernetes-release/release/v1.3.6/bin/linux/amd64/kubectl
+RUN chmod +x kubectl
+RUN mv kubectl /usr/local/bin/
+
+ADD gh-dl-release /bin/
+RUN chmod +x /bin/gh-dl-release
+
+ARG GITHUB_TOKEN
+ARG REPO=cloudposse/github-pam
+ARG FILE=github-pam_linux_386
+ARG VERSION=0.4
+
+RUN if [ ! -z $GITHUB_TOKEN ]; then \
```

Comment:
-n works different then ! -z
! -z return true only of env var exists and not empty